src/client/components/Home/RegisterButton.style.css

```diff
@@ -0,0 +1,18 @@
+.homepage-slogan{
+  height: 63vh;
+  width: 100vw;
+  text-align: center;
+  align-items: center;
+  color: white;
+  padding-top: 110px;
+  z-index: 2;
+}
+
+.homepage-slogan:nth-of-type(1){
+  background-color:  #6236FF;
```

Comment:
The bgcolor should be consider as this rgba(98, 54, 255, 0.63)

Reply:
this bgcolor  is mentioned in issue #102